src/main/java/io/github/bbortt/event/planner/web/rest/LocationResource.java

```diff
@@ -127,9 +128,9 @@ public LocationResource(LocationService locationService, ProjectService projectS
 
     @GetMapping("/locations/project/{projectId}")
     @PreAuthorize("@projectService.hasAccessToProject(#projectId, \"" + RolesConstants.ADMIN + "\", \"" + RolesConstants.SECRETARY + "\")")
-    public ResponseEntity<List<Location>> getLocationsByProjectId(@PathVariable Long projectId) {
+    public ResponseEntity<List<Location>> getLocationsByProjectId(@PathVariable Long projectId, Sort sort) {
```

Comment:
@zeroplexer you can actually just extend the existing API endpoint with a `Sort`. the default is `Sort.UNSORTED` which equals the database sort.

Reply:
Ah okey. Danke für den Hinweis.